tests/test_composite.py

```diff
@@ -40,56 +86,50 @@ def test_multi_block_init_vtk():
     assert isinstance(multi.GetBlock(2), pyvista.MultiBlock)
 
 
-def test_multi_block_init_dict():
-    data = dict()
-    data['grid'] = ex.load_rectilinear()
-    data['poly'] = ex.load_airplane()
+def test_multi_block_init_dict(rectilinear, airplane):
+    data = {'grid': rectilinear, 'poly': airplane}
     multi = pyvista.MultiBlock(data)
     assert isinstance(multi, pyvista.MultiBlock)
     assert multi.n_blocks == 2
     # Note that dictionaries do not maintain order
-    assert isinstance(multi.GetBlock(0), (pyvista.RectilinearGrid, pyvista.PolyData))
-    assert multi.get_block_name(0) in ['grid','poly']
-    assert isinstance(multi.GetBlock(1), (pyvista.RectilinearGrid, pyvista.PolyData))
+    assert type(multi.GetBlock(0) in (pyvista.RectilinearGrid, pyvista.PolyData))
+    assert multi.get_block_name(0) in ['grid', 'poly']
+    assert type(multi.GetBlock(1) in (pyvista.RectilinearGrid, pyvista.PolyData))
     assert multi.get_block_name(1) in ['grid','poly']
 
 
-def test_multi_block_keys():
-    data = dict()
-    data['grid'] = ex.load_rectilinear()
-    data['poly'] = ex.load_airplane()
+def test_multi_block_keys(rectilinear, airplane):
+    data = {'grid': rectilinear, 'poly': airplane}
     multi = pyvista.MultiBlock(data)
     assert len(multi.keys()) == 2
     assert 'grid' in multi.keys()
     assert 'poly' in multi.keys()
 
 
-def test_multi_block_init_list():
-    data = [ex.load_rectilinear(), ex.load_airplane()]
+def test_multi_block_init_list(rectilinear, airplane):
+    data = [rectilinear, airplane]
     multi = pyvista.MultiBlock(data)
     assert isinstance(multi, pyvista.MultiBlock)
     assert multi.n_blocks == 2
     assert isinstance(multi.GetBlock(0), pyvista.RectilinearGrid)
     assert isinstance(multi.GetBlock(1), pyvista.PolyData)
 
 
-def test_multi_block_append():
+def test_multi_block_append(ant, sphere, uniform, airplane, rectilinear):
     """This puts all of the example data objects into a a MultiBlock container"""
     multi = pyvista.MultiBlock()
     # Add examples
-    multi.append(ex.load_ant())
-    multi.append(ex.load_sphere())
-    multi.append(ex.load_uniform())
-    multi.append(ex.load_airplane())
-    multi.append(ex.load_rectilinear())
+    multi.append(ant)
+    multi.append(sphere)
+    multi.append(uniform)
+    multi.append(airplane)
+    multi.append(rectilinear)
     # Now check everything
     assert multi.n_blocks == 5
     assert multi.bounds is not None
-    assert isinstance(multi[0], pyvista.PolyData)
-    assert isinstance(multi[1], pyvista.PolyData)
-    assert isinstance(multi[2], pyvista.UniformGrid)
-    assert isinstance(multi[3], pyvista.PolyData)
-    assert isinstance(multi[4], pyvista.RectilinearGrid)
+    indices = range(5)
+    types = (PolyData, PolyData, UniformGrid, PolyData, RectilinearGrid)
+    assert all(type(multi[i]) == t for i, t in zip(indices, types))
```

Comment:
We should track the type of data being loaded in and then loop back through.  Make a `data` list with the potential data types, and then populate another `types` list with that type while it's being appended to `multi`.  Then, do assertions based on the len of `data`, not just 5 since we might change the length of data.

Reply:
Better? This should help assertion output and is shorter.

Reply:
What you have is an improvement, just suggesting that we not hardcode ``(PolyData, PolyData, UniformGrid, PolyData, RectilinearGrid)``, but what we have is good.

Reply:
No, I mean I committed a different version of this section:

    multi = MultiBlock()
    # Add and test examples
    datasets = (ant, sphere, uniform, airplane, rectilinear)
    for i, dataset in enumerate(datasets):
        multi.append(dataset)
        assert multi.n_blocks == i + 1
        assert isinstance(multi[i], type(dataset))